ezy-client.js

```diff
@@ -60,25 +73,44 @@ class EzyConnector {
         }
     }
 
+    /**
+     * Close the websocket connection with a reason
+     * @param {string} reason
+     */
     disconnect(reason) {
         if(this.ws) {
             this.disconnectReason = reason;
             this.ws.close();
         }
     }
 
+    /**
+     * Client purposely closes the websocket connection
+     */
     destroy() {
         this.destroyed = true;
         this.disconnect();
     }
 
+    /**
+     * Send data from client to websocket server
+     * @param {string} data
+     */
     send(data) {
         var json = JSON.stringify(data);
         this.ws.send(json);
     }
 }
 
+/**
+ * A central class that abstracts all the communications
```

Comment:
đây là một lớp proxy vậy nên là: A proxy class that combine all needed functions into one?